src/routes/terms/+page.markdoc

```diff
@@ -64,50 +57,73 @@ You agree not to disclose your password to any third party. You must notify us i
+- Using the Service in connection with any content, product, or service the recipient has not explicitly requested.
 
-We strongly advise you to read the terms and conditions and privacy policies of any third-party web sites or services that you visit.
+- Sending SMS messages through the Service unless you have obtained prior express consent from the recipients, as required by applicable law.
+
+Failure to adhere to these prohibitions and any applicable laws relating to the use of the SMS Service can result in immediate termination of your account and further legal action. You shall comply with all applicable laws and regulations.
+
+# Reporting Abuse
+
+If you become aware of any misuse of the Service, including any prohibited conduct or content, please report it immediately to us at **[abuse@appwrite.io](mailto:abuse@appwrite.io)**.
 
 # Termination
 
 We may terminate or suspend your account immediately, without prior notice or liability, for any reason whatsoever, including without limitation if you breach the Terms.
 
 Upon termination, your right to use the Service will immediately cease. If you wish to terminate your account, you may simply discontinue using the Service.
 
-# Limitation Of Liability
+# Limitation of Liability
+
+In no event shall Appwrite, nor its directors, employees, partners, agents, suppliers, or affiliates, be liable for any indirect, incidental, special, consequential, or punitive damages, including without limitation, loss of profits, data, use, goodwill, or other intangible losses, resulting from:
 
-In no event shall Appwrite, nor its directors, employees, partners, agents, suppliers, or affiliates, be liable for any indirect, incidental, special, consequential, or punitive damages, including without limitation, loss of profits, data, use, goodwill, or other intangible losses, resulting from (I) your access to or use of or inability to access or use the Service; (II) any conduct or content of any third party on the Service; (III) any content obtained from the Service; and (IV) unauthorized access, use, or alteration of your transmissions or content, whether based on warranty, contract, tort (including negligence) or any other legal theory, whether or not we have been informed of the possibility of such damage, and even if a remedy set forth herein is found to have failed of its essential purpose.
+- Your access to or use of or inability to access or use the Service.
 
+- Any conduct or content of any third party on the Service.
+
+- Any content obtained from the Service.
+
+- Unauthorized access, use, or alteration of your transmissions or content.
```

Comment:
To match the previous text, it would be good to use numbered lists. How do they look here?